tools/build-tools/data/layerInfo.json

```diff
@@ -152,17 +152,17 @@
             "HostUtils": {
                 "dev": true,
                 "packages": [
-                    "@fluid-experimental/get-container",
-                    "@fluid-experimental/fluid-static",
                     "@fluid-experimental/frs-client",
-                    "@fluid-experimental/tinylicious-client"
+                    "@fluid-experimental/tinylicious-client",
+                    "@fluid-experimental/get-container"
                 ],
                 "deps": [
                     "Loader",
                     "Routerlicious-Driver",
                     "Server-Libs",
                     "Server-Shared-Utils",
-                    "Test"
+                    "Test",
+                    "Framework"
```

Comment:
nit: alphabetize

Reply:
WAIT FOR PASSING BUILD :D

Reply:
Y U NO COMPILE FIRST? 😋

Reply:
i was able to build just fine, but had some linting and other errors...i just want a full CD pass before making a tweak like that

Reply:
AND WE GOT IT :D